<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerEdit</class>
 <widget class="QDialog" name="SellerEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>View My Products</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>581</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 13px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Product Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Category</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>470</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 127);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Product Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>470</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 127);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Product Quantity:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>510</y>
     <width>411</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>Edit Product Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>931</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt;
font: 22pt &quot;Snap ITC&quot;;
color: rgb(255, 255, 127);
font-weight: 900;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Welcome Seller , Edit Your Products Here</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
